full-service/src/json_rpc/v2/api/wallet.rs

```diff
@@ -172,7 +172,7 @@ where
             block_version,
             sender_memo_credential_subaddress_index,
             payment_request_id,
-            subaddress_to_spend_from,
+            spend_from_subaddress,
```

Comment:
Renaming from previous PR that introduced these. Assuming no need for backward compatibility